### PR TITLE
Python 3 and flake 8 fixes

### PR DESCRIPTION
**Pull request by Aniket21mathur**

### CODE REVIEW

**Comment by quozl** [comment]
I don't think BaseException is any different from leaving it out.

**Comment by Aniket21mathur** [comment]
> I don't think BaseException is any different from leaving it out.

Yes, agreed. Forced some changes please review.

Thanks!

File: clock.py [hunk 1]
```diff
@@ -185,8 +185,8 @@ def __init__(self, handle):
 
         # Some hardware cannot keep two GStreamer playback pipelines active
         try:
-            model = file('/proc/device-tree/openprom/model', 'r').readline()
-        except:
+            model = open('/proc/device-tree/openprom/model', 'r').readline()
+        except IOError:
```
Comment (rhl-bthr) [inline clock.py]: specs of `open()` function at https://docs.python.org/3/library/functions.html#open says,
> Open file and return a corresponding file object. If the file cannot be opened, an OSError is raised.

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
Changes requested

File: clock.py (hunk 1 above)
Comment (Aniket21mathur) [reply clock.py]: Thanks.Agreed. I made the changes accordingly please review.

**Comment by quozl** [comment]
Some of the code that failed flake8 was no longer required, so I rebased, rewrote and force pushed _master_.  https://github.com/sugarlabs/clock-activity/commit/66a0f267a2082d60ed12997080761800ceacdc45  https://github.com/sugarlabs/clock-activity/commit/313a63d97a9b6c2abe8d972213eb5fe60f47c31a.

**Pull request merged**